features/resending_credit_memo.feature

```diff
@@ -0,0 +1,26 @@
+@refunds
+Feature: Resending credit memo
+    In order to provide a generated credit memo to the customer
+    As an Administrator
+    I want to be able to send credit memo more that once
+
+    Background:
+        Given the store operates on a single green channel in "United States"
+        And the store has a product "Mr. Meeseeks T-Shirt" priced at "$10.00"
+        And the store has a product "Portal Gun" priced at "$20.00"
+        And the store allows shipping with "Galaxy Post"
+        And the store allows paying with "Space money"
+        And there is a customer "rick.sanchez@wubba-lubba-dub-dub.com" that placed an order "#00000022"
+        And the customer bought a single "Mr. Meeseeks T-Shirt"
+        And the customer bought a single "Portal Gun"
```

Comment:
Merge these two steps to `And the customer bought a "Mr. Meeseeks T-Shirt" and a "Portal Gun"`